tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -1138,12 +1141,11 @@ def test_dao_get_notifications_by_to_field_escapes(
     assert len(dao_get_notifications_by_to_field(
         sample_email_template.service_id,
         search_term,
-        notification_type='email',
+        notification_type=EMAIL_TYPE,
     )) == expected_result_count
 
 
 @pytest.mark.parametrize('search_term', [
-    '001',
```

Comment:
I don't think this test case is still valid.  "001" normalizes to "1", which matches both notifications below.